pulp_container/app/viewsets.py

```diff
@@ -586,7 +586,7 @@ class ContainerPushRepositoryViewSet(TagOperationsMixin, ReadOnlyRepositoryViewS
                 "principal": "authenticated",
                 "effect": "allow",
                 "condition": [
-                    "has_model_or_obj_perms:container.modify_content_containerrepository",
+                    "has_model_or_obj_perms:container.modify_content_containerpushrepository",
```

Comment:
should we add another condition to modify_content? without the view_repo permission you won't even be able to `get` it

```
$ http POST  :24817/pulp/api/v3/repositories/container/container-push/c62453c6-2216-4453-9e6e-bfe9cef43f6b/tag/ tag=lala digest='sha256:2c1099ea9644e527fa5f15401c4d9ba62dbf30aa70473e72dfde85450cd96dbd' --auth ina:goodpassword
HTTP/1.1 404 Not Found
Access-Control-Expose-Headers: Correlation-ID
Allow: POST, OPTIONS
Connection: close
Content-Length: 23
Content-Type: application/json
Correlation-ID: b458c1ba9830487c9466e603195d4ebe
Date: Fri, 29 Jan 2021 21:48:02 GMT
Server: gunicorn/20.0.4
Vary: Accept, Cookie
X-Frame-Options: SAMEORIGIN

{
    "detail": "Not found."
}

```

Reply:
this also stands got other repo actions like sync/copy/etc

Reply:
I don't think, we have any update-needs-view-permission in any of the other policies.

Reply:
I think it would be good to be explicit, so when users look up the policy they see which permissions are needed.